client/gutenberg/extensions/presets/jetpack/editor-shared/jetpack-plugin-sidebar.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import { createSlotFill } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+import { PluginSidebar, PluginSidebarMoreMenuItem } from '@wordpress/edit-post';
+import { registerPlugin } from '@wordpress/plugins';
+
+/**
+ * Internal dependencies
+ */
+import './jetpack-plugin-sidebar.scss';
+import JetpackLogo from 'components/jetpack-logo';
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+
+const { Fill, Slot } = createSlotFill( 'JetpackPluginSidebar' );
+
+const JetpackIcon = () => <JetpackLogo />;
+
+const JetpackPluginSidebar = ( { children } ) => (
+	<Fill>
+		{ children }
+	</Fill>
+);
+
+JetpackPluginSidebar.Slot = () => (
```

Comment:
The `Fill` / `Slot` functionality could be non-straightforward. Dev who're looking at this, I'd appreciate it if you tell me what was unclear in this file, so I could add some inline documentation.

Reply:
I'm not too familiar with the pattern, but it seems to be a standard util in Gutenberg. You could add a code comment to https://wordpress.org/gutenberg/handbook/components/slot-fill/, but OTOH, where do we draw the line when it comes to adding explicit links to WP docs? So maybe don't add a link :slightly_smiling_face:

Reply:
Thanks for verifying folks 👍